### DISCUSSION TITLE
GET /offers/listing z parametrem "numer oferty"

### DISCUSSION THREAD

**Discussion by k0rba** [allegro/allegro-api#4585] [Suggestions - rejected]
Skoro z zasobu [`GET /offers/listing`](https://developer.allegro.pl/documentation/#operation/getListing) mogą korzystać już tylko zweryfikowane aplikacje, chciałbym zaproponować dodanie do parametrów zapytania wyszukiwanie po numerze oferty. Parametr mógłby nawet być "wielokrotny", jak `include`. W moim przypadku zredukuje to transfer i ograniczy liczbę dużo bardziej kosztownych (jak mniemam) wyszukiwań z parametrem `phrase`.

**Comment by MartaNowaczyk** [comment]
Zaktualizowaliśmy status sugestii jako "odrzucona", ponieważ aktualnie zasobu GET /offers/listing nie ma w publicznym API.
